src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -177,6 +186,10 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
         }
     }
 
+    private void assignTopic(final EventType eventType) {
+        eventType.setTopic(uuidGenerator.randomUUID().toString());
```

Comment:
I thought about something like that. But topic name has some limitations which is already source of bugs:

1. underscores and dots conflicts
2. size limitation
3. other characters limitations